README.md

```diff
@@ -14,12 +14,43 @@ Create a workflow like the one below (also in example directory):
 ```
 
 NOTE:
-The PR commit message wallet address must be seperated by a space " " character from the message.
+The PR commit message wallet address must be seperated by a separator input (default: space) from the message.
```

Comment:
We should be able to do just the first `0x` without any regard for space right?

At least that is what we had done our [Solidity code](https://github.com/git-0consensus/contracts/blob/d12d71a378bbb437e9294430fbf17c86cda78cef/contracts/lib/Utils.sol#L24